### PR TITLE
Support Kafka streams

### PR DESCRIPTION
**Pull request by ilia-tsyplenkov**
Support Kafka streams at it was done for Redis

### CODE REVIEW

File: mindsdb/integrations/kafka/kafkadb.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+    def work(self):
+        self.consumer = kafka.KafkaConsumer(bootstrap_servers=f"{self.host}:{self.port}",
+                                            consumer_timeout_ms=1000)
+
+        self.consumer.subscribe([self.control_topic_name])
+        log.error(f"Integration {self.name}: subscribed  to {self.control_topic_name} kafka topic")
+        while not self.stop_event.wait(0.5):
+            try:
+                # break if no record about this integration has found in db
```
Comment (George3d6) [inline mindsdb/integrations/kafka/kafkadb.py:109]: If this comment is necessary, than the method should be called `integration_in_db`  or something similar, if the method is or will be checking for more than db presence, then remove this comment

File: mindsdb/integrations/kafka/kafkadb.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        self.streams = {}
+        self.stop_event = STOP_THREADS_EVENT
+
+    def should_i_exist(self):
+        config_record = session.query(Configuration).filter_by(company_id=self.company_id).first()
+        if config_record is None:
+            return False
+        integrations = json.loads(config_record.data)["integrations"]
+        if self.name not in integrations:
+            return False
```
Comment (George3d6) [inline mindsdb/integrations/kafka/kafkadb.py:46]: This check is being done the wrong way, please check the configuration object directly since that already does the merging of the file config with the db config.

If we can't access the config here then we are doing something wrong and the control flow should be changed, but I think we can.

**Review by George3d6 — CHANGES_REQUESTED** [review]

File: mindsdb/integrations/kafka/kafkadb.py [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+
+    def _query(self):
+        pass
+
+    def register_predictors(self, model_data_arr):
+        pass
+
+    def unregister_predictor(self, name):
+        pass
```
Comment (George3d6) [inline mindsdb/integrations/kafka/kafkadb.py:172]: Why are these here, why not just skip streaming integration in the respective checks? Or is it easier this way?

**Review by George3d6 — CHANGES_REQUESTED** [review]

File: mindsdb/integrations/kafka/kafkadb.py (hunk 3 above)
Comment (ilia-tsyplenkov) [reply mindsdb/integrations/kafka/kafkadb.py]: Fixed

File: mindsdb/integrations/kafka/kafkadb.py (hunk 1 above)
Comment (ilia-tsyplenkov) [reply mindsdb/integrations/kafka/kafkadb.py:64]: Renamed to `exist_in_db`

File: mindsdb/integrations/kafka/kafkadb.py (hunk 2 above)
Comment (ilia-tsyplenkov) [reply mindsdb/integrations/kafka/kafkadb.py]: Thanks. Changed on `Config()` usage
